bindings/java/src/org/sleuthkit/datamodel/BlackboardAttribute.java

```diff
@@ -518,8 +521,23 @@ public String getDisplayString() {
 				return Double.toString(getValueDouble());
 			case BYTE:
 				return bytesToHexString(getValueBytes());
-			case DATETIME:
-				return FsContent.epochToTime(getValueLong());
+				
+			case DATETIME: {
+				try {
+					final Content dataSource = getParentArtifact().getDataSource();
+					if ((dataSource != null) && (dataSource instanceof Image )) {
+						// return the date/time string in the timezone associated with the datasource,
+						Image  image = (Image) dataSource;
+						TimeZone tzone = TimeZone.getTimeZone(image.getTimeZone());
+						return epochToTime(getValueLong(), tzone);
+					} 
+				} catch (TskException ex) {
+					// contnue with TimeZone.getDefault();	
+				}
```

Comment:
I agree with this. But this is in Sleuthkit datamodel and I didn't find any logging in those modules. I didn't think this warranted throwing an exception. Lets discuss alternatives.